src/EditorFeatures/Core/Shared/Options/FeatureOnOffOptions.cs

```diff
@@ -83,6 +83,9 @@ internal static class FeatureOnOffOptions
         public static readonly Option2<int> UseEnhancedColors = new(
             nameof(FeatureOnOffOptions), nameof(UseEnhancedColors), defaultValue: 1,
             storageLocations: new RoamingProfileStorageLocation("WindowManagement.Options.UseEnhancedColorsForManagedLanguages"));
+
+        public static readonly PerLanguageOption2<bool> AddImportsOnPaste = new(
+            nameof(FeatureOnOffOptions), nameof(AddImportsOnPaste), defaultValue: true);
```

Comment:
if this is on by default, then i don't think we can be blocking.  If it's off by default, being blocking may be ok initially.

Reply:
Could do a feature rollout and see how people react.

Reply:
agreed, can we do feature rollout here and have both? Or at least a way to change it to disabled by default?